kevm-pyk/src/kontrol/__main__.py

```diff
@@ -516,6 +542,7 @@ def parse(s: str) -> list[T]:
         ],
     )
     foundry_show_args.add_argument('test', type=str, help='Display the CFG for this test.')
+    foundry_show_args.add_argument('--id', type=str, default=None, required=False, help='ID of the test')
     foundry_show_args.add_argument(
         '--omit-unstable-output',
         dest='omit_unstable_output',
```

Comment:
Let's factor out a `foundry_test_args` in `cli.py`, and add the arguments `'test'` and `--id` to it, then use that everywhere here that both arguments are used.

Reply:
Looks much better, thanks!